tests/integration/test_security_report.py

```diff
@@ -364,7 +364,7 @@ def test_adds_origin_header(mini_sentry, relay, json_fixture_provider):
         origin="http://valid.com",
     )
 
-    envelope = mini_sentry.captured_events.get(timeout=10)
+    envelope = mini_sentry.get_captured_event()
```

Comment:
For these is the plan to try them with 5 and if they fail to set them back to 10?